src/serializer/serializer.js

```diff
@@ -1874,16 +2010,21 @@ export class Serializer {
     // Serialize for the first time in order to gather reference counts
     if (!this.options.singlePass) {
       if (this.options.profile) console.time("[Profiling] Reference Counts Pass");
-      this.collectValToRefCountOnly = true;
-      this.valToRefCount = new Map();
+      this._init(
+        /*detectValuesUsedBySingleFunctionValue*/true,
```

Comment:
Possibly for clarity, replace arguments with an object `{detectValuesUsedBySingleFunctionValue: boolean, collectValToRefCountOnly: boolean}`?